packages/workbox-background-sync/Queue.mjs

```diff
@@ -190,10 +273,10 @@ class Queue {
       self.addEventListener('sync', (event) => {
         if (event.tag === `${TAG_PREFIX}:${this._name}`) {
           if (process.env.NODE_ENV !== 'production') {
-            logger.log(`Background sync for tag '${event.tag}'
-                has been received, starting replay now`);
+            logger.log(`Background sync for tag '${event.tag}'` +
+                `has been received`);
           }
-          event.waitUntil(this.replayRequests());
+          event.waitUntil(this._onSync(this));
```

Comment:
As per https://github.com/GoogleChrome/workbox/wiki/Styleguide#callbacks, can we use an object as the parameter, and change this call to `this._onSync({queue: this})`?

And then the callback itself would need to use destructing. I'd like to stay consistent (and I prefer that model of parameters anyway).

Reply:
Done.